webpack-intro/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "webpack-intro",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
```

Comment:
you only want one script property, you can have as many properties in there as you want!